kalite/distributed/management/commands/screenshots.py

```diff
@@ -0,0 +1,338 @@
+import errno
+import json
+import os
+import glob
+import re
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.core.urlresolvers import reverse
+from django.utils.six import StringIO
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+from fle_utils.general import ensure_dir
+
+from kalite.testing.base import KALiteBrowserTestCase
+from kalite.testing.mixins import FacilityMixins, BrowserActionMixins
+
+from optparse import make_option
+
+try:
+    from local_settings import *
+    import local_settings
+except ImportError:
+    local_settings = object()
```

Comment:
I think it was so we can override where the screenshots will be saved on our `local_settings.py` (I usually set temporary outputs to `~/temp/`), the image extension to use, and the database to use.  

Here are the settings I've found here:

* SCREENSHOTS_OUTPUT_PATH = os.path.join(os.path.realpath(PROJECT_PATH), "..", "data", "screenshots")
* SCREENSHOTS_EXTENSION = ".png"
* SCREENSHOTS_DATABASE_NAME = "screenshot-data.sqlite"

Not sure if they still apply after all the refactors I've seen here. :)